core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -51,6 +52,16 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
       "-disown"
     )
 
+    conf.sslConf match {
+      case Some(ssl) => {
+        cmdToLaunch ++ Array("-internal_security_config", ssl)
+        val sslConfig = new Properties()
+        sslConfig.load(new FileInputStream(ssl))
+        cmdToLaunch ++ Array("-files", sslConfig.get("h2o_ssl_jks_internal") + "," + sslConfig.get("h2o_ssl_jts"))
```

Comment:
Can we add hadoop argument ( `-files`) after bunch of application specific arguments ?